tools/cli/flags.go

```diff
@@ -485,6 +486,10 @@ func getFlagsForStart() []cli.Flag {
 			Name:  JitterStartSeconds,
 			Usage: "Optional workflow start jitter in seconds. If set, workflow start will be jittered between 0-n seconds (after delay)",
 		},
+		cli.StringFlag{
+			Name:  FirstRunAtTime,
+			Usage: "Optional workflow's first run start time. If set, workflow will start at the specified time.",
```

Comment:
it's not obvious what should be the unit/format of this flag. please update usage description

Reply:
Thanks, I have updated the format description.